test/parallel/test-http-should-keep-alive.js

```diff
@@ -41,34 +43,32 @@ const SHOULD_KEEP_ALIVE = [
   true,  // HTTP/1.1, Connection: keep-alive
   false  // HTTP/1.1, Connection: close
 ];
-let requests = 0;
-let responses = 0;
 http.globalAgent.maxSockets = 5;
 
+const countdown = new Countdown(SHOULD_KEEP_ALIVE.length , () => server.close());
+let countdownIndexInc = SERVER_RESPONSES.length - countdown.remaining;
```

Comment:
You won't be able to store this in a variable since it won't update. I suppose you could make it a function if you want to avoid having to repeat it everywhere... `const getCountdownIndex = () => SERVER_RESPONSES.length - countdown.remaining`  and then call it as `getCountdownIndex()`.